--- Custom/UILibrary/APButton.xaml.cs

+++ Upstream/UILibrary/APButton.xaml.cs

@@ -6,8 +6,9 @@

     public partial class APButton : System.Windows.Controls.UserControl
     {
-        public APButton(string Text, string? tooltip = null)
+        public APButton(string Text, string? tooltip = null, string iconGlyph = "\uE8B0")
         {
             InitializeComponent();
             ButtonTitle.Content = Text;
+            IconLabel.Content = iconGlyph;
 
             if (!string.IsNullOrEmpty(tooltip))