--- Custom/UISections/ModelMenuControl.xaml

+++ Upstream/UISections/ModelMenuControl.xaml

@@ -16,80 +16,5 @@

                          Height="33">
                     <TabItem.Header>
-                        <TextBlock Text="Model AI (Slot 1)"
-                                   FontSize="11"
-                                   Foreground="White"
-                                   FontFamily="{StaticResource Atkinson Hyperlegible}" />
-                    </TabItem.Header>
-                    <Grid>
-                        <ScrollViewer VerticalScrollBarVisibility="Visible"
-                                      PanningMode="VerticalOnly"
-                                      Margin="0,10,-6,0">
-                            <StackPanel>
-                                <Border Background="#3F3C3C3C"
-                                BorderBrush="#3FFFFFFF"
-                                BorderThickness="1"
-                                CornerRadius="5,5,0,0"
-                                Margin="0,0,0,0"
-                                Height="39">
-                                    <TextBox
-                                x:Name="Slot1ModelSearchBox"
-                                FontFamily="{StaticResource Atkinson Hyperlegible}"
-                                Padding="5"
-                                Foreground="White"
-                                CaretBrush="White"
-                                TextChanged="Slot1ModelSearchBox_TextChanged">
-                                    </TextBox>
-                                </Border>
-                            </StackPanel>
-                        </ScrollViewer>
-                        <Border Background="#3F3C3C3C"
-                                BorderBrush="#3FFFFFFF"
-                                BorderThickness="1"
-                                Margin="0,55,0,54">
-                            <ListBox x:Name="Slot1ModelListBox"
-                                     AllowDrop="True"
-                                     Drop="ModelListBox_Drop"
-                                     DragOver="ModelListBox_DragOver"
-                                     Foreground="#DDFFFFFF"
-                                     FontFamily="{StaticResource Atkinson Hyperlegible}"
-                                     Margin="0"
-                                     Padding="2">
-                            </ListBox>
-                        </Border>
-                        <Border Background="#3F3C3C3C"
-                                BorderBrush="#3FFFFFFF"
-                                BorderThickness="1"
-                                CornerRadius="0,0,0,5"
-                                Margin="0,0,44,10"
-                                VerticalAlignment="Bottom"
-                                Height="39">
-                            <Label x:Name="Slot1Notifier"
-                                   Foreground="#DDFFFFFF"
-                                   Content="Đã tải Slot 1: N/A"
-                                   FontFamily="{StaticResource Atkinson Hyperlegible}"
-                                   Padding="10"></Label>
-                        </Border>
-                        <Button Background="#3F3C3C3C"
-                                BorderBrush="#3FFFFFFF"
-                                Content="&#xE838;"
-                                HorizontalAlignment="Right"
-                                Height="39"
-                                Width="39"
-                                Margin="0,0,0,10"
-                                VerticalAlignment="Bottom"
-                                Padding="0,0,0,0"
-                                FontFamily="Segoe MDL2 Assets"
-                                BorderThickness="1"
-                                materialDesign:ButtonAssist.CornerRadius="0,0,5,0"
-                                FontSize="13"
-                                Foreground="White"
-                                Click="OpenFolderB_Click"
-                                Tag="models" />
-                    </Grid>
-                </TabItem>
-                <TabItem Style="{StaticResource ModelsTabItem}"
-                         Height="33">
-                    <TabItem.Header>
-                        <TextBlock Text="Model AI (Slot 2)"
+                        <TextBlock Text="Local Models"
                                    FontSize="11"
                                    Foreground="White"
@@ -143,5 +68,5 @@

                             <Label x:Name="SelectedModelNotifier"
                                    Foreground="#DDFFFFFF"
-                                   Content="Đã tải Slot 2: N/A"
+                                   Content="Loaded Model: N/A"
                                    FontFamily="{StaticResource Atkinson Hyperlegible}"
                                    Padding="10"></Label>
@@ -170,5 +95,5 @@

                          Width="90">
                     <TabItem.Header>
-                        <TextBlock Text="Configs Cục Bộ"
+                        <TextBlock Text="Local Configs"
                                    FontSize="11"
                                    Foreground="White"
@@ -221,5 +146,5 @@

                             <Label x:Name="SelectedConfigNotifier"
                                    Foreground="#DDFFFFFF"
-                                   Content="Đã tải Config: N/A"
+                                   Content="Loaded Config: N/A"
                                    FontFamily="{StaticResource Atkinson Hyperlegible}"
                                    Padding="10"></Label>
@@ -248,5 +173,5 @@

                          Width="132">
                     <TabItem.Header>
-                        <TextBlock Text="Tải Models"
+                        <TextBlock Text="Downloadable Models"
                                    FontSize="11"
                                    Foreground="White"
@@ -258,5 +183,5 @@

                                HorizontalContentAlignment="Center"
                                VerticalContentAlignment="Center"
-                               Content="Hiện không có models nào để tải, vui lòng kiểm tra lại sau."
+                               Content="There is no downloadable models at the moment, please check again later."
                                Foreground="#DDFFFFFF" />
                         <ScrollViewer x:Name="ModelStoreMenu"
@@ -289,5 +214,5 @@

                          Width="132">
                     <TabItem.Header>
-                        <TextBlock Text="Tải Configs"
+                        <TextBlock Text="Downloadable Configs"
                                    FontSize="11"
                                    Foreground="White"
@@ -299,5 +224,5 @@

                                HorizontalContentAlignment="Center"
                                VerticalContentAlignment="Center"
-                               Content="Hiện không có configs nào để tải, vui lòng kiểm tra lại sau."
+                               Content="There is no downloadable configs at the moment, please check again later."
                                Foreground="#DDFFFFFF" />
                         <ScrollViewer x:Name="ConfigStoreMenu"